src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+// helper fn
+function isMan(person) {
+  return person.sex === 'm';
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredTab = people
+    .filter(isMan)
+    .filter(haveDiedInCentury);
```

Comment:
It's better to use a single `filter` method instead of chaining two filters. You can combine the conditions using `&&` inside the `filter` callback. 🧐